models/evc.py

```diff
@@ -41,7 +41,9 @@ class EVCBase(GenericEntity):
         "queue_id",
         "sb_priority",
         "primary_constraints",
-        "secondary_constraints"
+        "secondary_constraints",
+        "uni_a",
+        "uni_z"
```

Comment:
```suggestion
        "uni_z",
```

Minor suggestion to make it easy in an eventual next modification.